docs/installation/development-installation.md

```diff
@@ -0,0 +1,90 @@
+## Development Installation
+
+### Prerequisites
+
+* Java JDK 14 or later
+* Maven 3
+* NodeJs (16.9.1) and NPM (8.3.2) - [NodeJS Install](https://nodejs.org/en/download/package-manager/)
+* mariaDB 10.6 (available for development via docker-compose scripts)
+* using Keycloak is optional
+
+If you need more information about installing prerequisites (ubuntu / linux mint), please check [here](prerequisites-installation.md)
+
+### Installation Steps
+
+:exclamation: Each step is executed from project home directory.
+
+1) go to deployment folder and start environment (database and keycloak) via docker-compose:
+
+    ```bash
+    cd deployment
+    docker-compose -f localenv-docker-compose.yml up
+    ```
+
+2) go to webclient/app and install ui
+
+    ```bash
+    cd webclient/app
+    npm install --legacy-peer-deps
+    ```
+
+3) build project
+
+    ```bash
+    mvn clean install -P frontend
```

Comment:
Is there the profile?

Reply:
ja - baust du das nicht damit???

Reply:
Ich wusste nur, dass neulich da drüber geredet hatten. Deswegen wollte ich nochmal nachfragen. Ich lasse das immer mit npm und dem Server laufen

Reply:
> Ich wusste nur, dass neulich da drüber geredet hatten. Deswegen wollte ich nochmal nachfragen. Ich lasse das immer mit npm und dem Server laufen

Das solltest du immer mitlaufen lassen. Mind. vor dem Erstellen des PRs. Es wird ja auch so deployt.